hw/top_earlgrey/data/top_earlgrey.sv.tpl

```diff
@@ -512,7 +519,12 @@ module top_${top["name"]} #(
     .host_req_done_o (flash_host_req_done),
     .host_rdata_o    (flash_host_rdata),
     .flash_ctrl_i    (${m["inter_signal_list"][0]["top_signame"]}_req),
-    .flash_ctrl_o    (${m["inter_signal_list"][0]["top_signame"]}_rsp)
+    .flash_ctrl_o    (${m["inter_signal_list"][0]["top_signame"]}_rsp),
+    .scanmode_i,
+    .scan_reset_ni   (scan_rst_ni),
+    .ast_eflash_i,
+    .flash_test_mode_ai,
+    .flash_test_voltage_hi
```

Comment:
should these suffixes have another `_`? I see below that other power indication signals are using `*_h_o`

Reply:
yeah that's what i realized they were doing, but i thought our style guide concatenates the suffixes?

Reply:
if this is meant to be in the sense of our suffixes [here](https://github.com/lowRISC/style-guides/blob/master/VerilogCodingStyle.md#suffixes-for-signals-and-types), you are right and these should be concatenated.

btwy, these chars are currently unused, so we could standardize these suffixes if we think that makes sense...